src/test/strategy.test.ts

```diff
@@ -0,0 +1,30 @@
+import { HttpRequestMessageConfig } from '../routing/Http/HttpRequestMessageConfig';
+import ProcessBodyStrategy from '../routing/Http/Strategies/ProcessBodyStrategy';
+import { Person } from '../services/apiTypes';
+
+describe('ProcessBodyStrategy is depended in a string that can be serialized', () => {
+    it('should fail if the configuration is not valid', () => {
```

Comment:
Denne testen tester ikke noe. Venter den på noe mer for å fullføres, eller er den bare ikke ferdigstilt?